learn/cookbooks/docker.md

```diff
@@ -0,0 +1,135 @@
+# Using Meilisearch with Docker
+
+A common way of installing Meilisearch is using Docker. In this guide you will learn what Docker is, how to install it in your local machine, and how to use Meilisearch with it.
+
+## Docker
+
+Docker is a tool that packages an application's code and dependencies into something called a container. Using Docker allows developers to ensure their software runs reliably across a wide range of machines and environments.
+
+Docker containers are distributed in images. Docker images are packages storing not only the container, but also instructions on how Docker should instantiate that container.
+
+## Install Docker
+
+The first step to start using Docker is installing it in your target machine. We recommend following [Docker's official instructions to install Docker Desktop](https://docs.docker.com/get-docker/), available for MacOS, Windows, and Linux.
+
+Docker for Desktop includes an interface for managing your containers, command-line applications and several tools required for using Docker.
+
+## Install Meilisearch
+
+Once you installed Docker you can use the `docker pull` command to download a Meilisearch image:
+
+```sh
+docker pull getmeili/meilisearch:v0.28
+```
+
+Meilisearch deploys a new Docker image for every new release. All images are tagged with the release version. You can see [the full list of available Meilisearch Docker images](https://hub.docker.com/r/getmeili/meilisearch/tags#!) on Docker Hub.
+
+::: warning
+Meilisearch advises against using the `latest` tag. As its name indicates, `latest` will always get the most recent Meilisearch release, which might result in different machines running different images if significant time passes between setting up each of them.
+:::
```

Comment:
Since docker run also pulls the image, is there really an advantage to explaining the pull command?

In general, I would say the more we reduce the number of step to get meilisearch running the better.

Reply:
Couldn't  using `run` without `pull`ing before  result in Docker not using the latest version of an image?

Reply:
Well, you still need to specify the version you want to use when you `run`, right?

Or is your point about multiple `run` over a long period of time?
Iirc `docker run` will first look in the cached images; if it doesn't need anything, it will run `docker pull` and then execute.
But if it does find something, it won't `pull` anything, and thus you might miss all the minor versions of meilisearch 🤔 

That's an interesting point; maybe we should keep the `docker pull` but move it later in the section to make the initial installation seems easier? 🤔 
We could make an « update meilisearch » which tells you to run `docker pull` and explain the dump or something like that.

But then the « casual » user who doesn't know much about docker is going to miss all the bug fixes of the minor version, which isn't what we want either 😬 

Honestly, I have no idea of what you should prioritize, but I think you have all the information to make a decision 😂